org.keystore_explorer.KeyStoreExplorer.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.keystore_explorer.KeyStoreExplorer
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: kse
+rename-desktop-file: kse.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+
+  - name: KeyStoreExplorer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p deb-package
+      - ar p KeyStoreExplorer.deb data.tar.gz | tar -xzf - -C deb-package
+      - mv deb-package/opt/kse/* /app/
+      - mkdir /app/bin/
+      - ln -s /app/kse.sh /app/bin/kse
+      - install -Dm644 org.keystore_explorer.KeyStoreExplorer.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 /app/icons/kse_512.png /app/share/icons/hicolor/512x512/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - install -Dm644 /app/icons/kse_256.png /app/share/icons/hicolor/256x256/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - install -Dm644 /app/icons/kse_128.png /app/share/icons/hicolor/128x128/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - install -Dm644 /app/icons/kse_48.png /app/share/icons/hicolor/48x48/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - install -Dm644 /app/icons/kse_32.png /app/share/icons/hicolor/32x32/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - install -Dm644 /app/icons/kse_16.png /app/share/icons/hicolor/16x16/apps/org.keystore_explorer.KeyStoreExplorer.png
+      - desktop-file-edit --set-icon=org.keystore_explorer.KeyStoreExplorer /app/kse.desktop
+      - desktop-file-install /app/kse.desktop --dir=/app/share/applications
+
+    sources:
+      - type: file
+        dest-filename: KeyStoreExplorer.deb
+        url: https://github.com/kaikramer/keystore-explorer/releases/download/v5.5.3/kse_5.5.3_all.deb
+        sha256: 59f3e161318ed717c2d717f38b0af1049a95e2eb7924e9ef6cf7ae3f6232f82b
```

Comment:
why not build from source?

Reply:
This is a Java project that uses gradle as a build tool, it requires interne to download the dependencies during build, not sure if there is a way to do that or not.

Reply:
Have you tried https://github.com/flatpak/flatpak-builder-tools/tree/master/gradle#flatpak-gradle-generator ? This is an example that uses it https://github.com/flathub/org.ghidra_sre.Ghidra/blob/master/org.ghidra_sre.Ghidra.json

Reply:
I have not tried it, I will give it a go.

Reply:
I was not successfull
- I was able to build locally and get the log file, but parsing it with [flatpak-builder-tools - gradle](https://github.com/flatpak/flatpak-builder-tools/tree/master/gradle) did not work, complaint about some URL from the log.
- Also tried this one with no luck [jwharm/flatpak-gradle-generator](https://github.com/jwharm/flatpak-gradle-generator)

I would like to continue with the `.deb` package if that is not a blocker, after all it's pulled directly from upstream.

Reply:
Ok